pydantic/dataclasses.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass as dc
+from .main import validate_model, create_model
+
+
+def pydantic_model(cls):
+    fields = {name: (field.type, field.default)
```

Comment:
Nitpick (sorry):

pydantic has 120 char line limit, I think you can put this on two lines.

Reply:
I have 80 char limit in my editor, do you mean the comprehension, which is split into 3 lines is too long? I change it to for loop.

Reply:
https://github.com/samuelcolvin/pydantic/blob/master/setup.cfg#L8 this projects uses 120, best to be consistent, but if it's difficult don't worry.

I'd prefer to keep it as a comprehension.